apps/src/music/views/TimelineSimple2Events.jsx

```diff
@@ -87,6 +87,7 @@ const TimelineSimple2Events = ({
       <div style={{position: 'absolute'}}>
         {uniqueFunctionExtents.map((uniqueFunction, index) => (
           <div
+            key={index}
```

Comment:
key (don't know what's inside uniqueFunction, not sure if it's possible to use anything else as a key )